databaseconnect.py

```diff
@@ -150,3 +150,54 @@ def learn_question_response(H):
     db.commit()
     B = "Thank you! I have learnt this."
     return B,0
+
+def clear_table(table_name):
+    db = connection_to_database()
+    cur = db.cursor()
+
+    if table_name in ("question_table","statement_table"):
+        tables_to_be_deleted = ["question_table","statement_table"]
+        print("The following tables will be deleted:\n")
+        for table in tables_to_be_deleted:
+            cur.execute("DESC %s" % table)
+            res = cur.fetchall()
+            column_names = [col[0] for col in res]
+
+            cur.execute("SELECT COUNT(*) FROM %s" % table)
+            res = cur.fetchall()
+            records_no = res[0][0]
+
+            print("Table Name:", table)
+            print("Columns:", column_names)
+            print("Number of existing records:", records_no)
+            print()
```

Comment:
On reviewing my own code, I find this part of code is repetitive. Do you think I should create a function for this?

Reply:
Perhaps, yes
That would make it look cleaner

Reply:
I don't think a function is needed for something like this. 
You could put it all in one line and put it in a variable before the conditional block since both set of statements use roughly the same variable names (small modification to the variable `table` required) - `disp_text=f'Table Name:{table}\nColumns:{column_names}\n'`.